test/end-to-end/pages/editRecipe.js

```diff
@@ -32,7 +32,7 @@ module.exports = class EditRecipePage extends MainPage {
     // Recipe Name link
     this.varLinkToViewRec = `${this.mailUrl}#/recipe/${this.recipeName}`;
     if (pageConfig.root.includes('9090')) {
-      this.varLinkToViewRec = `${this.mailUrl.slice(0, -6)}cockpit/@localhost/welder/index.html#/recipe/${this.recipeName}`;
+      this.varLinkToViewRec = `${this.mailUrl.slice(0, -10)}cockpit/@localhost/welder-web/index.html#/recipe/${this.recipeName}`;
```

Comment:
Same question as above, why is this necessary?

Reply:
The "welder-web" has ten characters, but welder has six.